unfoldingWord/translationCore
# Populate tH resources actions/reducers #1871

## This PR Addresses:

Populate tH resources actions/reducers

## To Test This PR:

- Checkout same branch from translationHelps, translationNotes, and translationWords
- Open existing project and translationNotes/translationWords
- Ensure tH displays articles from multiple menu groups.
- Open tH modal.
- Click on links inside of tH pane.
- Click on links inside of tH modal.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1876)
